src/app/agent/agent-dashboard/select-traveler/select-traveler.component.html

```diff
@@ -43,7 +43,7 @@
       <fieldset>
         <legend>Create a Traveler</legend>
         <p>
-          Create a new account for your traveler.
+          Upon completion of this form the traveler will complete the account creation process. You can still book flights for a traveler before they complete this step.
```

Comment:
Something like "...traveler will receive an email to complete..." would be more clear.